modules/ROOT/pages/configure-ingress-http-resource.adoc

```diff
@@ -0,0 +1,229 @@
+= Configuring Ingress with HTTP Route-Template Custom Resource in Runtime Fabric
+
+Configure ingress support for Mule apps on the Anypoint Runtime Fabric platform using the HTTP route-template custom resource.
+
+== Benefits of Ingress Configuration with the HTTP Route-Template Custom Resource
+
+Current ingress support for Mule apps on the Runtime Fabric platform is confined to https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes ingress] resources, which restrict you from using cloud vendor-native HTTP routing solutions, such as OpenShift Routes on OpenShift clusters, Istio, or Service Mesh. To address this limitation, Runtime Fabric supports a custom resource `httproutetemplates.rtf.mulesoft.com` template.
+
+Use the HTTP route-template custom resource in Runtime Fabric to define and implement custom routing configurations. With this approach, you can take advantage of compatibility with a range of ingress controllers and cloud-native routing mechanisms, including support for:
+
+* Wildcard for host-based routing
+* Creation of `httproutetemplates.rtf.mulesoft.com` templates to map a set of application endpoints to arbitrary HTTP routing resources for path-based routing
```

Comment:
The context is already set above so i would just keep it :
`Map a set of application endpoints to arbitrary HTTP routing resources for path-based routing`